src/clefnote.ts

```diff
@@ -60,9 +73,16 @@ export class ClefNote extends Note {
     this.setRendered();
     const abs_x = this.getAbsoluteX();
 
-    Glyph.renderGlyph(ctx, abs_x, stave.getYForLine(this.clef.line), this.clef.point, this.clef.code, {
-      category: `clefNote_${this.size}`,
-    });
+    Glyph.renderGlyph(
+      ctx,
+      abs_x,
+      stave.getYForLine(this.clef.line),
+      this.size == 'default' ? Tables.NOTATION_FONT_SCALE : (Tables.NOTATION_FONT_SCALE / 3) * 2,
```

Comment:
This same line of code appears a lot. And I'm not sure why it is multiplied by 2/3 in the non-default case.

Should we consolidate all of these `'default' ? Tables.NOTATION_FONT_SCALE : (Tables.NOTATION_FONT_SCALE / 3) * 2` into a helper method? If you don't think it's worth it, that's fine. Just curious.... because it shows up a lot.

Reply:
Done in `Clef.getPoint`